index.bs

```diff
@@ -143,7 +143,7 @@ The following is an informative summary of the format specifiers processed by th
 
 The printer operation is implementation-defined. It accepts a log level indicating severity, and a List of arguments to print (which are either JavaScript objects, of any type, or are implementation-specific representations of printable things such as a stack trace, or output produced by the <code>%o</code> and <code>%O</code> specifiers). How the implementation prints <var>args</var> is up to the implementation, but implementations should separate the objects by a space or something similar, as that has become a developer expectation.
```

Comment:
I think the list of things that arguments can contain (in parenthesis) needs to be expanded to include the group concept.